### PR TITLE
implemented input check against hex in message verify

### PR DESCRIPTION
**Pull request by fileninja**
as the title says. resolving #206

### CODE REVIEW

File: client/app/src/accounts/AccountService.js [hunk 1]
```diff
@@ -354,6 +354,12 @@
     };
 
     function verifyMessage(message, publicKey, signature) {
+      //check for hexadecimal, otherwise the signature check would may fail
+      var re = /[0-9A-Fa-f]{6}/g;
+      if(!re.test(publicKey) || !re.test(signature))
+      {
+        return;
```
Comment (fix) [inline client/app/src/accounts/AccountService.js]: should return false instead

File: client/app/src/accounts/AccountController.js [hunk 2]
```diff
@@ -1939,7 +1939,10 @@
         var res = accountService.verifyMessage(message, publickey, signature);
         $mdDialog.hide();
         var message = gettextCatalog.getString("Error in signature processing");
-        if (res == true) {
+        if(!res)
```
Comment (fix) [inline client/app/src/accounts/AccountController.js]: not needed

File: client/app/src/accounts/AccountService.js (hunk 1 above)
Comment (fileninja) [inline client/app/src/accounts/AccountService.js]: Yeah i also Thinked about this but you can difference between unverified or wrong Input

Comment (fix) [inline client/app/src/accounts/AccountService.js]: then i would rather return something else from the method.
like `{success:false, error:"error message detailing error"}`

Comment (fileninja) [inline client/app/src/accounts/AccountService.js]: Yeah that would be better indeed. I will do.

**Comment by fileninja** [comment]
here you go @fix

**Comment by luciorubeens** [comment]
+5

**Pull request merged**